### PR TITLE
 LPS-96291 When scrolling horizontally in a permissions popup, the role names should remain fixed to the left side.

### PR DESCRIPTION
**Pull request by travisrcory**
https://issues.liferay.com/browse/LPS-96291

### CODE REVIEW

**Comment by drewbrokke** [comment]
Just started reviewing :)

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/taglib/_search_container.scss [hunk 1]
```diff
@@ -88,6 +88,11 @@
 .lfr-description-column, .lfr-role-column {
 	max-width: 375px;
 	min-width: 375px;
+
+	// &.lfr-search-iterator-fixed-table-data {
+	// 	max-width: 225px;
+	// 	min-width: 225px;
+	// }
```
Comment (drewbrokke) [inline modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/taglib/_search_container.scss]: Was this intended to remain commented, or should we remove it?

Comment (travisrcory) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/taglib/_search_container.scss]: Oh man, good catch, yes it should be removed

**Comment by johnrobhartzler** [comment]
@travisrcory @drewbrokke has this been merged yet, or are we waiting on some additional steps?

**Comment by drewbrokke** [comment]
@johnrobhartzler it's currently in review with Jonathan Mak at https://github.com/jonmak08/liferay-portal/pull/2088

**Pull request closed without merging**